base/applications/games/solitaire/lang/ro-RO.rc

```diff
@@ -56,7 +56,7 @@ END
 STRINGTABLE
 BEGIN
     IDS_SOL_NAME "Pasiență"
-    IDS_SOL_ABOUT "Pasiență de J Brown\nTradus de Petru Dimitriu\nCardLib versiunea 1.0."
+    IDS_SOL_ABOUT "Pasiență de J Brown; Tradus de Petru Dimitriu\nCardLib versiunea 1.0."
```

Comment:
The shell about box doesn't display more than two lines in the about-text?

Reply:
Correct. It looks like it should support three lines, but it cuts off the third line in ROS.